js/App.vue

```diff
@@ -8,9 +8,9 @@
     >
         <div class="text-center">
             <h1 class="text-6xl md:text-8xl mb-4">
-                <a href="/"> Holy Gambit Grail ! </a>
+                <a href="/">Holy Gambit Grail!</a>
             </h1>
-            <p class="md:text-2xl">How many of these gambits have you won with ?</p>
+            <p class="md:text-2xl">How close are you to completing the Holy Gambit Grail?</p>
```

Comment:
I don't want to repeat the word Holy Gambit Grail twice. The title already indicates what we are talking about. The second phrase defines what the holy gambit grail is about for a large audience.

Reply:
Ok. I agree. How about "How many gambits have you won with?"

Reply:
Ok with that one.